stable/nats/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: nats
-version: 1.2.0
+version: 1.2.1
```

Comment:
This PR breaks backwards compatibility. Therefore, it should increase the major version of the chart.

Reply:
I reverted the `podAnnotations` -> `annotations` change, the PR should now no longer contain breaking changes. I'll bump the minor, as the changes are more than bug fixes. Please let me know if this is ok.

Reply:
cool! Thanks